src/utils/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const gameReviewAPI = axios.create({
+  baseURL: "https://jasmines-nc-games.onrender.com/api",
+});
+
+function fetchReviews() {
+  return gameReviewAPI
+    .get("/reviews", {})
+    .then((res) => {
+      return res.data;
+    })
+    .catch((err) => {
```

Comment:
At a later point you can handle this error in your component. For now it's fine.